pe/pe.py

```diff
@@ -404,6 +404,36 @@ def recurse_resources(node, parent_name):
             res.add_lines(dataless_resources)
             self.file_res.add_section(res)
 
+    # Inspired by https://github.com/CYB3RMX/Qu1cksc0pe/blob/086db196d2de289f0784ae4d8ee03f34bf10354b/Modules/winAnalyzer.py#L446
+    def check_number_of_functions(self):
```

Comment:
I think this could be added to the add_headers() function, before the `self.features["nx"] = self.binary.has_nx` where I am already referencing those attributes.

Reply:
https://github.com/CybercentreCanada/assemblyline-service-pe/pull/54/commits/56acd64f3f98b28f9ccd9d32681fe2399a2eddb3